docs/src/projects/configuration.md

```diff
@@ -105,6 +105,28 @@ response: [0x1, 0x23, 0x4]
 </details>
 <br>
 
+### Global configuration
+
+Since version 0.34.0 `sncast` will use the global configuration to combine configuration from local configs.
+Global configuration file is usual [`snfoundry.toml`](https://foundry-rs.github.io/starknet-foundry/appendix/snfoundry-toml.html), but placed in specific directory. 
+
+#### Global Configuration File Location
+The global configuration is stored in a specific location depending on the operating system:
+
+- macOS/Linux : The global configuration file is located at `$HOME/.config/starknet-foundry/snfoundry.toml`
+- Windows : The file can be found at `C:\Users\<user>\AppData\Roaming\starknet-foundry\snfoundry.toml`
+
+> 📝 **Note**
+> Global configuration file will be created automatically if it does not exist and contain predefined `default` profile.
+
+
+#### Interaction between Local and Global profiles
+
+Interaction between global and local config is based on the overriding mechanism, the local config will be used to override 
+the global profile of the same name or in case it is not defined to override the default global profile.
+
+If `--profile` is not provided, the default profile from the local (if present) and global configuration will be used.
```

Comment:
This needs to be written in simpler terms, so it's obvious what overrides what and in which cases.